docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -642,6 +642,7 @@ To install Gremlin-Python, use Python's link:https://en.wikipedia.org/wiki/Pip_(
 [source,bash]
 ----
 pip install gremlinpython
+pip install gremlinpython[kerberos]     # Optional, not available on Microsoft Windows
```

Comment:
I was trying to understand this syntax, specifically the addition of the bracketed "[kerberos]". I see it described pretty nicely here:

https://stackoverflow.com/a/46775606/1831717

that led me to this link:

https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

should "kerberos" be defined in `extra_require"?

Reply:
Sure, in fact it is on line 83 of the setup.py of gremlin-python.

Reply:
sorry - i don't know how i missed that.......